src/FEValues/common_values.jl

```diff
@@ -228,13 +230,15 @@ function function_divergence(fe_v::ScalarValues{dim}, q_point::Int, u::AbstractV
     return diverg
 end
 
-function function_divergence(fe_v::VectorValues{dim}, q_point::Int, u::AbstractVector{T}, dof_range::UnitRange = 1:length(u)) where {dim,T}
+function function_divergence(fe_v::VectorValues{dim}, q_point::Int, u::AbstractVector{T}, dof_range = eachindex(u)) where {dim,T}
     n_base_funcs = getn_scalarbasefunctions(fe_v)*dim
     @assert length(dof_range) == n_base_funcs
     @boundscheck checkbounds(u, dof_range)
     diverg = zero(T)
-    @inbounds for (i, j) in enumerate(dof_range)
-        grad = shape_gradient(fe_v, q_point, i)
+    basefunc_count = 1
+    @inbounds for j in dof_range
```

Comment:
`enumerate` is used in some other places too, but did this actually make a difference with the other changes?

Reply:
Not really, just made the code slightly smaller to look at.